server/src/models/threshold_model.ts

```diff
@@ -12,7 +12,8 @@ export enum Condition {
   ABOVE_OR_EQUAL = '>=',
   EQUAL = '=',
   LESS_OR_EQUAL = '<=',
-  LESS = '<'
+  LESS = '<',
+  NO_DATA = 'No data'
```

Comment:
```suggestion
  NO_DATA = 'NO_DATA'
```
similar to https://github.com/hastic/hastic-server/blob/master/server/src/models/analytics_task_model.ts#L10